sources/packages/web/nginx.conf

```diff
@@ -0,0 +1,30 @@
+user  nginx;
+worker_processes  1;
```

Comment:
We may need to review those configurations. Leaving this as the default value 1 can maybe lead to an underutilization of the available resources.

Reply:
We can use `auto` that will config the server to have the same number as available cores for now.